From 904e5637d990450c540574a6b87d411f30cf06d7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sat, 23 Mar 2024 13:07:37 +0800
Subject: [PATCH 08/13] fix(check.mjs): do not check Mihomo (Clash Meta)
 platform support

This is pointless, as we supply our own.
---
 scripts/check.mjs | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/scripts/check.mjs b/scripts/check.mjs
index 87b10c8..8390e2c 100644
--- a/scripts/check.mjs
+++ b/scripts/check.mjs
@@ -145,21 +145,6 @@ async function getLatestReleaseVersion() {
   }
 }
 
-/*
- * check available
- */
-if (!META_MAP[`${platform}-${arch}`]) {
-  throw new Error(
-    `clash meta alpha unsupported platform "${platform}-${arch}"`
-  );
-}
-
-if (!META_ALPHA_MAP[`${platform}-${arch}`]) {
-  throw new Error(
-    `clash meta alpha unsupported platform "${platform}-${arch}"`
-  );
-}
-
 /**
  * core info
  */
-- 
2.45.2

